docs/modules/operation/pages/operation/daemon-config-files/eventd.adoc

```diff
@@ -11,10 +11,25 @@
 
 .Eventd configuration file overview
```

Comment:
Will conflict with https://github.com/OpenNMS/opennms/pull/3591 from @mfuhrmann

Reply:
@mfuhrmann @mmahacek I would vote to merge this PR first and adjust PR #3591 afterward accordingly and not worry about the merge conflict here.